Model/Rate/Result.php

```diff
@@ -227,14 +225,7 @@ private function createTitle($settingPath)
     private function createPrice($alias, $settingPath) {
```

Comment:
De method heeft maar 1 regel nodig:
`return $this->myParcelHelper->getMethodPrice($settingPath . 'fee', $alias)`

Reply:
Done!